### PR TITLE
3) Other Pages

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+/* footer */
+.page-footer {
+    grid-area: p_footer;
+}
+
+@media only screen and (max-width: 855px) {
+    .page-wrap {
+    grid-auto-rows: 15rem auto auto 7.5rem;
+    }
+}
```
Comment (deekshithmd) [inline src/styles.css]: It looks good

**Review by deekshithmd — APPROVED** [review]

File: src/styles.css (hunk 1 above)
Comment (varun13169) [reply src/styles.css]: Thanks.

File: src/pages/productListingPage/product-listing-page.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  width: 100%;
+  height: 7rem;
+  background-color: var(--dui-light-theme-bg);
+  box-shadow: 0px 0px 0px 2px var(--dui-primary-color-p2-lgt-35pct);
+
+  position: fixed;
+  bottom: 0;
+  left: 0;
+
+  display: flex;
```
Comment (MayankKumar10) [inline src/pages/productListingPage/product-listing-page.css]: make helper class for flex

File: src/pages/authPage/auth-page.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+}
+
+
+/*TODO: move to component library */
+.dui-auth-card {
+    width: fit-content;
+    height: fit-content;
+    padding: 2rem;
+    padding-left: 7.5rem;
```
Comment (MayankKumar10) [inline src/pages/authPage/auth-page.css:33]: use shorthand property for padding and margin i.e Top,Right,Bottom,Left

File: src/pages/cartPage/cart-page.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+.page-wrap {
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content';
+}
+
+.page-sidebar {
+    display: none;
```
Comment (MayankKumar10) [inline src/pages/cartPage/cart-page.css:11]: make a class for display none

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestions Everything else looks good to me.

File: src/pages/cartPage/cart-page.css (hunk 4 above)
Comment (varun13169) [reply src/pages/cartPage/cart-page.css:11]: Not required, for a single property.

File: src/pages/productListingPage/product-listing-page.css (hunk 2 above)
Comment (varun13169) [reply src/pages/productListingPage/product-listing-page.css]: Not required, for a single property.

File: src/pages/authPage/auth-page.css (hunk 3 above)
Comment (varun13169) [reply src/pages/authPage/auth-page.css:33]: will be doing it as part of refactoring.

**Pull request merged**